modular_skyrat/code/datums/traits/neutral.dm

```diff
@@ -8,14 +8,14 @@
 	var/list/blacklistedspecies = list(/datum/species/synth, /datum/species/android, /datum/species/ipc, /datum/species/synthliz, /datum/species/shadow, /datum/species/plasmaman, /datum/species/jelly, /datum/species/jelly/slime)
 
 /datum/quirk/synthetic/add()
+	. = QDEL_IN(src, 120)
+	sleep(5 SECONDS)
```

Comment:
What's it needed for?

Reply:
Just to be sure that the species won't be gained too soon and cause limb icon problems, since getting the species later had proven to avoid the issue entirely.